### PR TITLE
add readme file.

### PR DESCRIPTION
**Pull request by florianziemen**
A minimal readme file.

Please suggest where more content is necessary.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+## Quarto
+
+The web pages are built using https://quarto.org See [their Get Started section](https://quarto.org/docs/get-started/) and the [guide](https://quarto.org/docs/guide/) for hints on usage.
+
+As quarto uses markdown, you can use pretty much any online pad, for prototyping of simple texts / bullet lists / ... with a live preview.
+
+## Github.com and branches
+
+To add to the repo, you can either [create a fork of this repository](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks), edit there, and then [make a pull request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork), or ask for direct access to this repo, and create a branch for your edits there.
```
Comment (lkluft) [inline README.md:13]: Maybe we should only advertise the fork-workflow here. If people ask for direct access themselves, we can still grant that (on a case-to-case basis)

File: README.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# Contributing 
+
+This repository is just for editing the page. The finished version is rendered at https://digital-earths-global-hackathon.github.io/planning/ 
+
+## Quarto
+
+The web pages are built using https://quarto.org See [their Get Started section](https://quarto.org/docs/get-started/) and the [guide](https://quarto.org/docs/guide/) for hints on usage.
```
Comment (lkluft) [inline README.md:7]:
```suggestion
The web pages are built using [quarto](https://quarto.org). See [their Get Started section](https://quarto.org/docs/get-started/) and the [guide](https://quarto.org/docs/guide/) for hints on usage.
```

**Review by lkluft — APPROVED** [review]
Thanks for adding this! 👍 I made some minor comments, but would be fine with merging as is 🚀

File: README.md (hunk 1 above)
Comment (florianziemen) [reply README.md]: done, thanks!

**Pull request merged**